bluetooth_mesh/interfaces.py

```diff
@@ -138,12 +137,38 @@ def __init__(self, element):
         super().__init__(name="org.bluez.mesh.Element1")
 
     @dbus_property(name="Models", access=PropertyAccess.READ)
-    def get_models(self) -> "aq":
-        return self.element.models
+    def get_models(self) -> "a(qa{sv})":
+        models = []
+
+        for model_id, publish, subscribe in self.element.models:
+            models.append(
+                [
```

Comment:
Nit: Make this a tuple instead of a list

Reply:
`dbus-next` requires `list` for then D-Bus objects `()` ;/